geode-core/src/main/java/org/apache/geode/internal/cache/execute/ServerToClientFunctionResultSender.java

```diff
@@ -321,16 +325,8 @@ public synchronized void setException(Throwable exception) {
           }
           String exceptionMessage = exception.getMessage() != null ? exception.getMessage()
               : "Exception occurred during function execution";
-          if (AbstractExecution.SUPPRESS_FUNCTION_EXCEPTION_LOGGING
-              && exception instanceof FunctionException) {
-            if (logger.isDebugEnabled()) {
-              logger.debug(String.format("Exception on server while executing function : %s",
-                  fn), exception);
-            }
-          } else {
-            logger.warn(String.format("Exception on server while executing function : %s",
-                fn), exception);
-          }
+          logger.warn(exception instanceof FunctionException ? FUNCTION_EXCEPTION_MARKER : null,
+              String.format("Exception on server while executing function : %s", fn), exception);
```

Comment:
Replace the `String.format()` with the logger format.

Reply:
Updated